tensorflow/lite/kernels/internal/kernel_utils.cc

```diff
@@ -137,6 +137,16 @@ void RnnBatchStep(
                hidden_state_ptr_batch, output_ptr_batch);
 }
 
+#define TFLITE_QUANTIZE_GET_MIN_MAX(units, batch, quantized_batch, scale) \
```

Comment:
Do you have data that suggests this macro is faster than an inlined function? Overall I think this is good to get rid of code duplication, but would only go this route if we had evidence that an inline function here was slower.